tailwind.config.cjs

```diff
@@ -1,14 +1,21 @@
-/** @type {import('tailwindcss').Config} */
 module.exports = {
   content: [
     "./index.html",
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: '#001529',
+        secondary: '#1e40af',
+        list1: '#0047cc',
```

Comment:
do we need this one? I mean we don't need the content provider from ant?

Reply:
@imadiaz I think so, but I could replace it with the function to change the color with ant, I used the taildwind.config file to modify the login texts, the backArrow and the names of the doctors in the schedules (they are in blue) and This way only the taildwind file is modified but I can use a function like in the other files (only that when the color was changed it must be changed in those three files)
![centralizado de colores11](https://github.com/cDentalCareGroup/web-cdentalcaregroup/assets/83984948/ac4d3149-05eb-49fc-aa3b-b61a1ae47f66)
![centralizado de colores12](https://github.com/cDentalCareGroup/web-cdentalcaregroup/assets/83984948/0cb30db2-1289-4399-8cbb-08753e7ba691)

Reply:
@imadiaz I checked it and it works fine
![centralizado de colores7](https://github.com/cDentalCareGroup/web-cdentalcaregroup/assets/83984948/f59944fa-a333-40dc-bcbc-9dff384ad13b)
![centralizado de colores8](https://github.com/cDentalCareGroup/web-cdentalcaregroup/assets/83984948/e29c27c5-bf54-4b9b-9387-2ce9c5d03b4e)
![centralizado de colores9](https://github.com/cDentalCareGroup/web-cdentalcaregroup/assets/83984948/9efbdf79-92a2-4dd9-ab0a-e708cae2ae7b)
![centralizado de colores10](https://github.com/cDentalCareGroup/web-cdentalcaregroup/assets/83984948/8b12f7a8-b1ca-4372-a52e-fcfc00556bdd)

Reply:
Thanks for this!